src/views/v2/commerce/success.php

```diff
@@ -29,7 +29,7 @@
 		<?php $this->template( 'order/header' ); ?>
 		<?php $this->template( 'order/description' ); ?>
 		<?php $this->template( 'order/details' ); ?>
-		<?php $this->template( 'order/attendees' ); ?>
+		<?php tribe( 'tickets.main' )->get_components_template()->template( 'attendees-list/attendees', $this->get_global_values(), true ); ?>
```

Comment:
```suggestion
		<?php tribe( 'tickets.editor.template' )->template( 'attendees-list/attendees', $this->get_global_values(), true ); ?>
```
I am pretty sure that it should be working with just `tribe( 'tickets.editor.template' )`, this would not require adding a new method for this.

Reply:
Thanks for the suggestion, I switched it to use that one instead.